TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/ResourceManagement.rst

```diff
@@ -81,6 +81,22 @@ publishing the resources and providing public URIs to it. From the name you can
 symlinks to the resources stored on the local filesystem to save space. Other ``Target`` implementations
 could publish the resources to CDNs or other external locations that are publicly accessible.
 
+If you have lots of resources in your project you might run into problems when executing ./flow resource:publish since the number of folders can be limited depending on the file system you're using. An error that might occur in this case is "Could not create directory".
```

Comment:
I've wrapped commands and paths. @albe I tried to include line breaks but they aren't shown. If I add two linebreaks there's a new paragraph but that's not what I wanted. Do you see what I did wrong in the last commit?